src/modules/common/components/amountField/Heading/Heading.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { TertiaryButton } from 'src/theme/buttons';
+import Icon from 'src/theme/Icon';
+import styles from './Heading.css';
+
+const Heading = ({ title, onGoBack, history, noBackButton, className }) => (
+  <div className={`${styles.wrapper} ${className}`}>
+    {!noBackButton && (
```

Comment:
this would be tricky because if `onGoBack` isn't a function what it means is that i would always have to provide onGoBack as a function for every simple goback functionality which can easily be handled in the component been wrapped with `withRouter`